code/src/pages/dashboard/dashboard-products/DashboardProductsPage.tsx

```diff
@@ -6,15 +8,31 @@ import ProductsTable from "@/containers/tables/products-table/ProductsTable";
 
 import AppBox from "@/components/app-box/AppBox";
 import AppButton from "@/components/app-button/AppButton";
+import AppContainer from "@/components/app-container/AppContainer";
+import AppPagination from "@/components/app-pagination/AppPagination";
 import AppTypography from "@/components/app-typography/AppTypography";
 
 import routePaths from "@/constants/routes";
+import { useLocaleContext } from "@/context/i18n/I18nProvider";
 import { useGetManagerProductsQuery } from "@/store/api/productsApi";
+import validatePage from "@/utils/validate-page/validatePage";
 
 import "@/pages/dashboard/dashboard-products/DashboardProductsPage.scss";
 
 const DashboardProductsPage = () => {
-  const { data, isLoading, error } = useGetManagerProductsQuery({});
+  const { locale } = useLocaleContext();
+
+  const [searchParams] = useSearchParams();
+
+  const searchParamsPage = searchParams.get("page");
+  const page = validatePage(searchParamsPage);
+
+  const { data, isLoading, error } = useGetManagerProductsQuery({
+    page: page - 1,
+    size: 8,
+    lang: locale,
+    sort: ["createdAt,desc"]
```

Comment:
I suggest creating some constants for sort options, which will include sort variants

Reply:
Currently we don't have sorting functionality on that page, so that is just a default value. I think such constant with all of the variants can be added when we add sorting on that page.